code/src/index.css

```diff
@@ -11,3 +12,175 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.movie-list-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.movie-card-container {
+  width: 100%;
+  position: relative;
+  text-decoration: none;
+}
+
+.poster-image {
+  width: 100%;
+  display: block;
+}
+
+.movie-card-container .image-overlay-text{
+  display:none;
+  position: absolute;
+  font-size: 20px;
+  color: white;
+  width: 100%;
+  height: 100%;
+  bottom: 0;
+  background: rgba(0,0,0,0.75);
+}
+
+.movie-card-container .image-overlay-text h1{
+  margin-top: auto;
+  margin-top: 0;
+}
+
+.movie-card-container:hover .image-overlay-text {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.image-overlay-text h1, h2 {
+  margin-bottom: 0;
+  padding: 10px;
+}
+
+.image-overlay-text h2 {
+  font-size: 16px;
+  font-weight:500;
+}
+
+/* Movie details page */
+
+.background-image {
+  height: 150vh;
+  width: 100%;
+  object-fit: cover;
+  filter: brightness(0.3);
```

Comment:
This was really clever of you, it helped make the page more readable in the summary of the movie! 🌟 We're gonna steal this later.